libraries/AP_Motors/AP_MotorsMulticopter.cpp

```diff
@@ -649,6 +663,12 @@ void AP_MotorsMulticopter::output_logic()
             break;
         }
 
+        // check whether we successfully got to spoolup
+        if (!check_failsafe()) {
+            AP::logger().Write_Error(LogErrorSubsystem::MOTORS, LogErrorCode::FAILSAFE_OCCURRED);
```

Comment:
need to ensure we don't log scary errors if the user has not configured ESC telemetry

Reply:
Fixed